feature/platform/transceiver/ZR_pre-fec_ber_test/README.md

```diff
@@ -0,0 +1,49 @@
+# gNMI-1.20: Telemetry: 400ZR Optics Pre-FEC(Forward Error Correction) BER(Bit Error Rate)
+
+## Summary
+
+Validate 400ZR optics module reports pre-FEC bit error rate performance data.
+
+## Procedure
+
+*   Connect two ZR interfaces using a duplex LC fiber jumper such that TX
+    output power of one is the RX input power of the other module.
+*   To establish a point to point ZR link ensure the following:
+      * Both transceivers state is enabled
+      * Both transceivers are set to a valid target TX output power
+        example -10 dBm
+      * Both transceivers are tuned to a valid centre frequency
+        example 193.1 THz
+*   With the link ZR link established as explained above, verify that the
```

Comment:
Is there any expected range of values?  Or just verify that the path exists and any value is returned?

Reply:
For the TX output power and centre frequency, we need to set these to fix values as called out.